electron-utils/oauth.ts

```diff
@@ -74,6 +76,7 @@ function getAuthorizationCode(parentWindow: BrowserWindow, repoPermissionLevel:
     });
 
     authWindow.webContents.on('will-redirect', (event, newUrl) => {
+      Logger.info('Redirecting to Github for authentication');
```

Comment:
I see, thank  you for the clarification!

Reply:
I actually have a question regarding this:
![image](https://user-images.githubusercontent.com/66008784/111054991-25e5f180-84ac-11eb-8750-630ce3aca6d9.png)

So we are calling "onCallback" for both redirect and navigate events, why do we need it for the "will-nagivate" event?

Reply:
I believe this is to handle the manual navigation back to our app, that occurs when a user authorises CATcher for the first time.

When a user tries to log-into CATcher for the first time, they would see:
![image](https://user-images.githubusercontent.com/35621759/111055308-993d3280-84af-11eb-9fc3-08be26d650c6.png)

Clicking the `Authorize CATcher-org` button would trigger the `will-navigate` event, rather than the automatic redirect.
@JunWei96 can I confirm if this is right?